<?xml version="1.0"?>
<launch> 
    <!-- 参考线平滑的方式选择: true:CosThetaSmoother  false:FemPosSmooth-->
    <arg name="which_smoother" default="false"/>    
    <param name="which_smoothers" value="$(arg which_smoother)"/>

    <!-- 规划的方式选择: 0:lattice  1:emplanner-->
    <arg name="which_planner" value="0"/>    
    <param name="which_planners" value="$(arg which_planner)"/>

    <!-- 是否使用曲率约束 -->
    <arg name="curvature_constraint" default="false"/>    
    <param name="apply_curvature_constraint" value="$(arg curvature_constraint)"/>
    <!-- 是否使用二次规划 -->
    <arg name="use_lateral_optimization" default="false"/> 

    <!-- ui界面 -->        
    <!-- <node pkg="ui_console" type="ui_console" name="interface_console" output="screen" /> -->
     <!-- 全局规划 -->
    <node pkg="planning" type="planning_node" name="planning" output="screen"/>
    <param name="use_sim_time" value="false"/>
     <!-- rviz -->
    <!-- <node pkg="rviz" type="rviz" name="rviz" args="-d $(find global_routing)/config/planning_demo.rviz"/> -->
 
</launch>
